tests/BaselineSearch/test_baseline_search.py

```diff
@@ -13,7 +13,7 @@ def run_test_get_preprocessed_stack_params(product):
 
     original_properties = product['properties']
     
-    assert(params.processingLevel == [get_default_product_type(product['properties']['sceneName'])])
+    assert(params.processingLevel == [get_default_product_type(reference)])
```

Comment:
We'll revisit in the future if we want to even have a default for stacking or just always match the ref, as previously mentioned.